### PR TITLE
Join table

### PR DESCRIPTION
**Pull request by Fadahunsiseyi**
- Understand the different types of relationships between tables.
- Prepare complex queries that answer analytical questions.
- Use primary key & foreign key mechanisms

Screenshot (333)
Screenshot (334)
Screenshot (335)
Screenshot (336)
Screenshot (337)
Screenshot (338)
 for joining tables.

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -20,6 +20,28 @@ ALTER TABLE species ADD PRIMARY KEY(id);
...
+vet_clinic(#     vet_id int references vets(id),
+vet_clinic(#     species_id int references species(id)
+vet_clinic(# );
+CREATE TABLE
+
+vet_clinic=# create table visits (
+vet_clinic(# vet_id int references vets(id),
+vet_clinic(# animals_id int references animals(id),
+vet_clinic(# date_of_visit date
+vet_clinic(# );
```
Comment (AnselemOdims) [inline schema.sql]: - [ ] Please kindly include only the queries instead of the terminal prompt

File: queries.sql [hunk 2]
```diff
@@ -169,4 +169,122 @@ vet_clinic-# animals.owner_id = owners.id group by owners.full_name order by cou
...
+ 2021-01-11    | Blossom | William Tatcher
+(1 row)
+
+
+vet_clinic=# select vets.name, species.name from vets
+vet_clinic-# left join specializations on specializations.vet_id = vets.id
+vet_clinic-# left join species on specializations.species_id = species.id;
+       name       |                         name
+------------------+------------------------------------------------------
+ William Tatcher  | Pokemon
```
Comment (AnselemOdims) [inline queries.sql]: - [ ] Good job working on this, but please as stated above kindly include only the queries and not the terminal prompt and output

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @Fadahunsiseyi , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: queries.sql (hunk 2 above)
Comment (Fadahunsiseyi) [reply queries.sql]: @AnselemOdims I have removed all of the command prompt outputs.

**Review by muneebulrehman — APPROVED** [review]
Hi @Fadahunsiseyi , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Good follow-up with the previous reviewer.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Fadahunsiseyi** [comment]
@muneebulrehman Thank you, but my dashboard is still saying changes required.

**Comment by muneebulrehman** [comment]
> @muneebulrehman Thank you, but my dashboard is still saying changes required.

That is because I requested changes in accordance with the previous reviewer. Maybe i mistakenly clicked on Approve in github but as you can see in the summary, it says "Required changes"

**Comment by Fadahunsiseyi** [comment]
@muneebulrehman I have removed all of the terminal prompts. Can you please highlight what you're referring to?

**Comment by Fadahunsiseyi** [comment]
> I have removed all of the terminal prompts. Can you please highlight what you're referring to?

 I have removed all of the terminal prompts. Can you please highlight what you're referring to?

**Comment by Fadahunsiseyi** [comment]
> > @muneebulrehman Thank you, but my dashboard is still saying changes required.
> 
> That is because I requested changes in accordance with the previous reviewer. Maybe i mistakenly clicked on Approve in github but as you can see in the summary, it says "Required changes"

 I have removed all of the terminal prompts. Can you please highlight what you're referring to?

**Review by elmar8287 — APPROVED** [review]
Hi, @Fadahunsiseyi.

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

### Highlights

- Done with queries 👍 

### Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Fadahunsiseyi** [comment]
@elmar8287 Thank you.

**Pull request merged**